test/mock/CMakeLists.txt

```diff
@@ -78,13 +91,15 @@ add_custom_target(
   COMMAND ${CMAKE_COMMAND} -E make_directory ${TEXTURES_BINARY_DIR}
   COMMAND ${CMAKE_COMMAND} -E make_directory ${CITADEL_BINARY_DIR}
   COMMAND ${CMAKE_COMMAND} -E make_directory ${CITADEL_OUTSIDE_BINARY_DIR}
+  COMMAND ${CMAKE_COMMAND} -E make_directory ${CITADEL_ENGINE_BINARY_DIR}
```

Comment:
In order to work in the run tests, there must be at least one file in here

Reply:
What should I add here? A python script gen a zzz file?